TortolaniHW03.Rmd

```diff
@@ -0,0 +1,158 @@
+  annotate("rect", xmin = as.Date("2020-03-25"), xmax = as.Date("2020-04-01"),
+                   ymin = 11250, ymax = 11750,
+                   fill = "firebrick3")
+
+#show plot
+RI_plot
+
+```
+
+
+##Graph 2
+
+Compare Rhode Island to Massachusetts.
+```{r}
+#filter the data for only Rhode Island
+all_states %>%
+  filter(state == "Massachusetts" | state == "Rhode Island") -> MA_RI_data
+  
+#re-format the date col
+MA_RI_data$date <- as.Date(MA_RI_data$date)
+
+#create the plot 
+MA_RI_plot <- ggplot(MA_RI_data) +
+  geom_col(aes(date, cases), fill = "dodgerblue3") +
+  geom_col(aes(date, deaths), fill = "firebrick3") +
+  facet_wrap(~state) +
+  
+#add axis labels / title
+  labs (title = "Total Number of COVID-19 Cases in Massachusetts vs Rhode Island",
+            x = "Date (year / month / day)", 
+            y = "Total Number of Cases / Deaths") +
+
+#make easier to read
+  theme(axis.text.x = element_text(angle=90, vjust = 0.65),
+        plot.title = element_text(size = 14, face = "bold", hjust = 0.5)) +
+  scale_x_date(date_breaks = "1 weeks") +
+  
+#create legend
+  annotate("text", as.Date("2020-02-24"), 60000, label = "Cases =") +
+  annotate("rect", xmin = as.Date("2020-03-16"), xmax = as.Date("2020-03-29"),
+                   ymin = 61000, ymax = 58000,
+                   fill = "dodgerblue3") +
+  annotate("text", as.Date("2020-02-24"), 45000, label = "Deaths =") +
+  annotate("rect", xmin = as.Date("2020-03-16"), xmax = as.Date("2020-03-29"),
+                   ymin = 46000, ymax = 43000,
+                   fill = "firebrick3")
+
+#show plot
+MA_RI_plot
+```
+
+
+##Graph 3
+
+Look at number of new daily cases in Rhode Island.
+```{r}
+#define new variables
+RI_data$new_cases <- lead(RI_data$cases, 1) - RI_data$cases
+RI_data$new_deaths <- lead(RI_data$deaths, 1) - RI_data$deaths
+
+#create the plot 
+RI_new_daily_plot <- ggplot(RI_data) +
+  geom_col(aes(date, new_cases), fill = "dodgerblue3") +
+  geom_col(aes(date, new_deaths), fill = "firebrick3") +
+  
+#add axis labels / title
+  labs (title = "Daily Number of COVID-19 Cases in Rhode Island",
+            x = "Date (year / month / day)", 
+            y = "Daily Number of Cases / Deaths") +
+
+#make easier to read
+  theme(axis.text.x = element_text(angle=90, vjust = 0.65),
+        plot.title = element_text(size = 14, face = "bold", hjust = 0.5),
+        legend.position = c(1,1)) +
+  scale_x_date(date_breaks = "1 weeks") +
+  
+#create legend
+  annotate("text", as.Date("2020-03-09"), 300, label = "Cases =") +
+  annotate("rect", xmin = as.Date("2020-03-20"), xmax = as.Date("2020-03-26"),
+                   ymin = 310, ymax = 290,
+                   fill = "dodgerblue3") +
+  annotate("text", as.Date("2020-03-09"), 250, label = "Deaths =") +
+  annotate("rect", xmin = as.Date("2020-03-20"), xmax = as.Date("2020-03-26"),
+                   ymin = 260, ymax = 240,
+                   fill = "firebrick3")
+
+#show plot
```

Comment:
After reading through your codes, I must compliment that you write them in a very clear way. I especially like how you state the aim of each chunk of codes like #make easier to read, #create legend etc. It makes it very easy to read and understand what you were trying to do. I shall try to make this a habit of mine as well when coding!

Reply:
Thanks!  In one of my courses when I was first learning how to code we were required to write a comment on every line to prove that it was a useful and meaningful line of code to include, although a bit excessive at times I think starting with this mentality really helped me.  We also were expected to hand write code for exams without the use of a computer so that definitely helped with organization.  If you are looking for ways to build your skills these tips definitely helped me when starting out!